less/components/notification.less

```diff
@@ -10,15 +10,15 @@
  */
 
 .cmp_notification {
-	background-color: @pinkish-grey;
+	background-color: @light-grey;
 	color: white;
 	padding: 7px;
 	border-radius: 5px;
 	text-align: center;
 	margin-bottom: 15px;
 
 	& a {
-		color: @black;
+		color: #000
```

Comment:
Can you check all semicolons in this PR?

Reply:
I seemed to have forgot to force-push this correction earlier - mea culpa - but all the semicolons should be there, otherwise the CSS wouldn’t have been compiled